docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -374,24 +271,22 @@ Selecting whole sub-object:
 SELECT (organization).office FROM users
 ----
 
-NOTE: When selecting the entire object, the query will always try to return the underlying object verbatim.
-For Java Types, this means returning an underlying Java class instance, which can fail with a `ClassNotFoundException`
-if the class is not in the classpath of the client (or embedded server) JVM.
-A way to avoid this is to select field by field instead. Additionally, this issue is not relevant for Compact
-and Portable types as sub-objects in these mappings and types are of `GenericRecord` subclass;
-`PortableGenericRecord` and `CompactGenericRecord` are present in the base distribution of Hazelcast.
+[NOTE]
+====
+. When selecting the entire object, the query will always try to return the underlying object verbatim. For Java-serialized types, this means returning an underlying Java class instance, which can fail with a `ClassNotFoundException` if the class is not in the classpath of the client (or embedded server) JVM. To avoid this, you can select individual fields instead. This issue does not apply to Portable- or Compact-serialized types as sub-objects in these mappings and types are `GenericRecord` subclasses; `PortableGenericRecord` and `CompactGenericRecord` are present in the base distribution of Hazelcast.
```

Comment:
I disagree. My usage of commas is correct.

Reply:
Keep the comma after types, then. I altered it because the sentence becomes very choppy when you use the other comma that I added. 

That would be the comma before "which", which is correct. If you prefer not to add the comma, change "which" to "that"

I was trying to do a really light-touch review this time, as you obviously take offence at every suggestion. However, if I was to have been picky, I would have pointed out that this sentence contains 41 words and needs to be re-written to address this.

Reply:
90767538